<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBoxSpec</class>
 <widget class="QGroupBox" name="GroupBoxSpec">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Ocean Spectrometer </string>
  </property>
  <widget class="QLineEdit" name="serial_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>27</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSerial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>27</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Serial:</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderIt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>86</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelIt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>59</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Integration Time:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="it_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>122</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelIt_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelIt_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>85</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>max</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetIt">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>421</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Plot">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkStreaming">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Streaming data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save_Aquisition">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filenametext">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelIt_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Filename:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="it_text_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelIt_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Measure Number:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditA">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_min">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_min_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Max:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
